ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -6,11 +6,20 @@
 }
 
 .pf-topology-resizable-side-bar {
-    /* we use the page header min height thrice to account for: nav header + page header + secondary header */ 
+    /* we use the page header min height thrice to account for: nav header + page header + secondary header */
     /* @TODO: Consider a more future-proof way of doing this. See https://github.com/stackrox/stackrox/pull/3703#discussion_r1014292340 */
     height: calc(100vh - var(--pf-c-page__header--MinHeight) - var(--pf-c-page__header--MinHeight) - var(--pf-c-page__header--MinHeight));
 }
 
+/* Create a stacking context that is higher than the context for the react-topology component
+   that is rendered below the toolbar, so that the search filter dropdown will render above the graph when open */
+[data-testid="network-graph-toolbar"] {
+    z-index: 150;
+}
+[data-testid="network-graph-toolbar"] .pf-search-shim,
+[data-testid="network-graph-toolbar"] .react-select__menu {
+    z-index: 150;
+}
```

Comment:
I think this does work for the topology component, but seems like the side panel still shows above it

<img width="716" alt="Screenshot 2023-01-18 at 1 54 42 PM" src="https://user-images.githubusercontent.com/4805485/213303949-38b9315e-b33a-42ab-a907-ab50488d6a35.png">

Reply:
@sachaudh 
Pushed a change to fix this
![Screen Shot 2023-01-19 at 12 08 35 PM](https://user-images.githubusercontent.com/715729/213512931-8d0802ec-6b6d-4182-bbf8-0337c3fe93f5.png)